lib/controllers/coinstac/consortia.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const config = require('config'); // jshint ignore:line
+const _ = require('lodash');
+const consortiaDbOptions = config.get('coinstac.pouchdb.consortia');
+
+const internals = {};
+
+internals.consortiaSchema = joi.object().keys({
+    label: joi.string().required(),
+    users: joi.array().required(),
+    description: joi.string().required(),
+    tags: joi.array().required(),
+    analyses: joi.array().required()
+});
+
+internals.savedConsortiaSchema = internals.consortiaSchema.keys({
+    _id: joi.string().required(),
+    _rev: joi.string().required()
+});
+
+internals.createConsortiaDb = (name, PouchW) => {
+    const defaults = consortiaDbOptions;
+    let pouchWConfig;
+
+    //TODO: return name of db for ease of location
+    name = defaults.name + '-' + _.kebabCase(name.toLowerCase());
+    if (consortiaDbOptions.path) {
+        pouchWConfig = {
+            path: defaults.path,
+            name: name
+        };
+    } else if (consortiaDbOptions.conn) {
+        pouchWConfig = {
+            name: name,
+            conn: defaults.conn
+        };
+    }
+
+    // create new db
+    const newDb = new PouchW(pouchWConfig);
+    return newDb;
+};
+
+module.exports.getAll = {
+    tags: ['api', 'coinstac', 'consortia'],
+    notes: [
+        'Returns all consortia stored in consortia DB',
+        'No query params supported currently'
+    ].join('<br>'),
+    description: 'Returns array of all consortia',
+    validate: {
+        query: joi.object().keys({})
```

Comment:
Good question.
Because this route does not accept any query params. Enforcing that query be empty achieves two things:

1. The swagger docs will state that no query params are accepted.
1. A request with query params will fail, thus alerting the client developer that their query params are not used by this endpoint. If we simply ignored their query params, then they would need to empirically test whether their query params had any effect on the output.